packages/teleport/src/DesktopSession/useDesktopSession.tsx

```diff
@@ -42,14 +43,55 @@ export default function useDesktopSession(ctx: Ctx) {
 
   const { username, desktopName, clusterId } = useParams<UrlDesktopParams>();
   const [hostname, setHostname] = useState<string>('');
+
+  const isUsingChrome = navigator.userAgent.indexOf('Chrome') > -1;
+  const clipboardEnabled = ctx.storeUser.getClipboardAccess();
+  const clipboardRWPermission = useClipboardRW(clipboardEnabled, 'prompt');
```

Comment:
No, it hadn't been properly tested by me. It should be fixed now.

Reply:
Because this is a hook with an internal state, we can't call it conditionally. So it must be called on non-chrome browsers, but must account for that fact. The updated `useClipboardPermission` now has an `isChrome` flag to account for this.